src/home/index.html

```diff
@@ -52,5 +53,170 @@ <h1 class="heading">I am Mwafrika Glad to see you!</h1>
         </ul>
       </div>
     </section>
+    <section class="works">
+      <section class="tonic">
+        <img src="../assert/images/tonic1.png" alt="" />
+        <h1>Tonic</h1>
+        <div class="tonic-list">
+          <div class="black">CANOPY</div>
+          <div class="dot"></div>
+          <div class="gray">Back End Dev</div>
+          <div class="dot"></div>
+          <div class="gray">2015</div>
+        </div>
+        <p class="tonicDesc">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="skills">
+          <p>html</p>
+          <p>css</p>
+          <p>javascript</p>
+        </div>
+        <div class="btn">
+          <button class="btn-tonic">See Project</button>

```

Comment:
Please implement the button interactions (enable, hover, etc.) for all the buttons in the "works" section. You can check [this link](https://www.w3schools.com/csSref/sel_hover.asp) for more information.

Reply:
Changed in CSS file. There's no javascript implementation so far.

Reply:
Hover is now working fine,

Reply:
![btn](https://user-images.githubusercontent.com/25848053/148602419-62e43a8a-201e-495f-895a-746cf2194ce8.png)

The button changes its colour when hover it.

Reply:
Addressed 🔥😎